Tests/RxCocoaTests/NSTextView+RxTests.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NSTextView+RxTests.swift
+//  Tests
+//
+//  Created by Cee on 8/5/18.
+//  Copyright © 2018 Krunoslav Zaher. All rights reserved.
+//
+
+import RxSwift
+import RxCocoa
+import AppKit
+import XCTest
+
+final class NSTextViewTests: RxTest {
+    /// NSTextView is much more complicated than other NS-prefix views or controls,
+    /// which takes a longer time to go through the `onCompleted` block. Here, we are
+    /// using `wait(for:timeout:)` to check dealloc/complete operations.
+    static let timeout: TimeInterval = 0.5
+
+    static let notificationForwardTestStrings: [String] = ["T", "Te", "Tes", "Test"]
+    static let notificationForwardTestAssertStrings: [String] = [""] + NSTextViewTests.notificationForwardTestStrings
```

Comment:
This should be moved to `testTextView_TextDidChange_ForwardsToDelegates` and removed `AssertStrings` and `Strings` part from the identifier name since they are redundant.